blocks/color-palette/index.js

```diff
@@ -16,7 +16,7 @@ import { __, sprintf } from '@wordpress/i18n';
  */
 import './style.scss';
 
-export function ColorPalette( { defaultColors, colors, value, onChange } ) {
+export function ColorPalette( { defaultColors, colors, value, onChange, allowCustom } ) {
```

Comment:
Do you think we should use `true` as a default value here to avoid having to update the components already using this?